core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java

```diff
@@ -186,7 +185,7 @@ public FlowIdentifier stopInstancesForClusterName(@ResourceName String clusterNa
     }
 
     @Override
-    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)
+    @InternalOnly
```

Comment:
since this logic does not need account id, you need to add `@AccountIdNotNeeded`
in 99% of the cases, internal APIs need a parameter (account id or initiator user crn or a resource crn) to help CB to decide which account are we in and which account we need to use in service layer

Reply:
https://github.com/hortonworks/cloudbreak/tree/master/authorization-common#how-to-make-an-api-internally-callable